test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -37,6 +38,35 @@ func StatefulSetIsReady(mdb *mdbv1.MongoDB) func(t *testing.T) {
 	}
 }
 
+func StatefulSetHasOwnerReference(mdb *mdbv1.MongoDB) func(t *testing.T) {
+	return func(t *testing.T) {
+		sts := appsv1.StatefulSet{}
+		err := f.Global.Client.Get(context.TODO(), types.NamespacedName{Name: mdb.Name, Namespace: f.Global.OperatorNamespace}, &sts)
+		if err != nil {
+			t.Fatal(fmt.Errorf("%s", err))
```

Comment:
in this case there's no real difference between `fmt.Errorf("%s", err)` and `err`

we should just be able to do `t.Fatal(err)`